cypress/e2e/tests/administration/jira-connection/interlinked.test.ts

```diff
@@ -0,0 +1,152 @@
+/*
+Copyright © 2021 the Konveyor Contributors (https://konveyor.io/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+/// <reference types="cypress" />
+
+import {
+    createMultipleApplications,
+    deleteByList,
+    exists,
+    login,
+    performRowAction,
+    validateTextPresence,
+} from "../../../../utils/utils";
+import { getJiraConnectionData, getJiraCredentialData } from "../../../../utils/data_utils";
+import {
+    cantDeleteJiraAlert,
+    CredentialType,
+    deleteAction,
+    JiraIssueTypes,
+    JiraType,
+} from "../../../types/constants";
+import { JiraConnectionData } from "../../../types/types";
+import { Jira } from "../../../models/administration/jira-connection/jira";
+import { JiraCredentials } from "../../../models/administration/credentials/JiraCredentials";
+import { MigrationWave } from "../../../models/migration/migration-waves/migration-wave";
+import * as data from "../../../../utils/data_utils";
+import { Assessment } from "../../../models/migration/applicationinventory/assessment";
+import { jiraAlert, jiraTable } from "../../../views/jira.view";
+
```

Comment:
@ibragins , Could you please add a comment with the Polarion test IDs ?

Reply:
These test cases should be independent of cloud or datacenter account . They can test on any account .
I have edited the TCs.

Reply:
OK, but when running test - I need to create this or that type of credentials. Which of them should I use? Or I can test on both, iterating between them